jobs/involuntary-dissolutions/involuntary_dissolutions.py

```diff
@@ -73,7 +77,46 @@ def shell_context():
     app.shell_context_processor(shell_context)
 
 
-def initiate_dissolution_process(app: Flask):  # pylint: disable=redefined-outer-name,too-many-locals
+def create_invountary_dissolution_filing(business_id: int):
+    """Create a filing entry to represent an involuntary dissolution filing."""
+    filing = Filing()
+    filing.business_id = business_id
+    filing.filing_type = CoreFiling.FilingTypes.DISSOLUTION
+    filing.filing_sub_type = DissolutionTypes.INVOLUNTARY
+    filing.save()
```

Comment:
This is the simplest filing for involuntary dissolution I could think of. Not sure if we need to include more information in the filing or if this is okay for now.

Reply:
We'll need to populated `filing.filing_json` for sure.  The dissolution processor looks at it.

Will take a look and see if we need anything else populated in the `filings` table.

Reply:
Yeah you're right, the `filing_json` was needed for the entity filer to process it, and it seems to be working now. Other than that, let me know if there is anything else we should be adding.

Reply:
was looking at an admin dissolution filing and I think populating the `filing_date` and `source` field would be good too.

Example admin dissolution filing in dev:
![image](https://github.com/bcgov/lear/assets/6685223/1e0fde71-e799-4d27-9dad-47b99b1ada62)